prompt.txt

```diff
@@ -0,0 +1,62 @@
+Tu es un inspecteur d'étiquette d'engrais qui travaille à l'agence d'inspection alimentaire canadienne. Mise en contexte : un texte extrait d'une image t'est fournie.
+Ton but est de classer l'entièreté des informations présente sans les modifier grâce à la liste de clé suivante.
+Tu dois absolument utiliser les clés dans la liste suivante.
+Ta réponse est exacte et non généré par LLM.
+Ta réponse est sous le format JSON.
+Ta réponse contient tout le texte présent dans le texte fourni.
+Avant de rendre ta réponse, vérifie celle-ci afin qu'elle concorde avec tous les points précédents.
+Voici la liste de clés :
+"compagny_name";
+"compagny_address";
+"compagny_website";
+"compagny_phone_number";
+"manufacturer_name";
+"manufacturer_address";
+"manufacturer_website";
+"manufacturer_phone_number";
+"fertiliser_name";
+"fertiliser_registration_number";
+"fertiliser_lot_number";
+"fertiliser_NPK";
+"fertiliser_precautionary_FR";
+"fertiliser_precautionary_EN";
+"fertiliser_instructions_FR";
+"fertiliser_instructions_EN";
+"fertiliser_ingredients_FR";
+"fertiliser_ingredients_EN";
+"fertiliser_specifications_FR";
+"fertiliser_specifications_EN";
+"fertiliser_cautions_FR";
+"fertiliser_cautions_EN";
+"fertiliser_recommendation_FR";
+"fertiliser_recommendation_EN";
+"fertiliser_first_aid_FR";
+"fertiliser_first_aid_EN";
+"fertiliser_warranty_FR";
+"fertiliser_warranty_EN";
+"fertiliser_danger_FR";
+"fertiliser_danger_EN";
+"fertiliser_guaranteed_analysis", sous-section "nutrient_in_guaranteed_analysis",  sous-section "percentage_in_guaranteed_analysis";
```

Comment:
this results in two fields. what we actually want is the nutrient as a key and the percentage as the value. something like this:
```json
"fertilizer_guarantee_analysis": {
  "Potasium": "90%",
  "Zinc": "5%",
}
```
or a table
```json
"fertilizer_guarantee_analysis": [
  ["Potasium", "90%"],
  ["Zinc", "5%"],
]
```

Reply:
Table is better